modules/ROOT/pages/apply-configure-json-threat-task.adoc

```diff
@@ -9,17 +9,23 @@ endif::[]
 >s|Policy Name|JSON Threat Protection
 >s|Summary|Protects against malicious JSON in API requests
 >s|Category|Security
->s|Since Mule Version|3.8.0
+>s|Available from Mule version|3.8.0
 >s|Returned Status Codes|400 - Bad Request
 |===
 
-== Purpose
+JSON requests are susceptible to attacks characterized by unusual inflation of elements and nesting levels. Attackers use recursive techniques to consume memory resources. Dramatic swings in the size of the application data often signal a security problem. The JSON Threat Protection policy helps protect your applications from such intrusions.
 
-JSON requests are susceptible to attacks characterized by unusual inflation of elements and nesting levels. Attackers use recursive techniques to consume memory resources. Dramatic swings in the size of the application data often signal a security problem. The JSON threat protection policy helps protect your applications from such intrusions.
+If Mule runtime engine (Mule) fails to detect an attack, you must monitor and design your services architecture with layers of protection in addition to these policies.
 
-In the event Mule Runtime fails to detect an attack, you need to monitor and design your services architecture with layers of protection in addition to these policies.
+== How This Policy Works
 
-== Configuration
+//anything to add here -- nahuel
+
+//which policies--nahuel
+
+== Configuring Policy Parameters
+
+When you apply the policy to your API from the UI, the following parameters are displayed:
 
 [width="100%", cols="2,6,1,1"]
 |===
```

Comment:
Default values on the table are incorrect. The default is not "0" but "-1".

Reply:
Done! Thanks!